python/.pylintrc

```diff
@@ -18,6 +18,7 @@ disable=missing-module-docstring,
         too-few-public-methods,
         invalid-name,
         fixme,
+        duplicate-code,
```

Comment:
Why is this needed?

Reply:
It thought imports were duplicate code. I find this kind of linting pretty annoying, so disabled it globally